conf/messages.en

```diff
@@ -176,7 +176,16 @@ xmlNegativeAcknowledgement.errorType = Error type
 xmlNegativeAcknowledgement.errorPointer = Error pointer
 xmlNegativeAcknowledgement.errorReason = Error reason
 xmlNegativeAcknowledgement.originalAttributeValue = Original attribute value
-xmlNegativeAcknowledgement.create.unloadingRemarks = create new unloading remarks
+
+unloadingRemarksNegativeAcknowledgement.title = There is a problem with these unloading remarks
```

Comment:
Can the xmlNegativeAcknowledgement message keys be removed or are they being used elsewhere?

Reply:
They're being used elsewhere